Assets/Localization/StringTables/Internal_RSC_en.asset

```diff
@@ -23791,13 +23791,13 @@ MonoBehaviour:
     m_Metadata:
       m_Items: []
   - m_Id: 1289
-    m_Localized: 'Perserverance Day is quite a party in Ykalon. It was[/center]
+    m_Localized: 'Perseverance Day is quite a party in Ykalon. It was[/center]
 
       originally
       held as a solemn memorial to those killed[/center]
 
       in battle, resisting
-      the Camaron Usurper, but has[/center]
+      the Camoron Usurper, but has[/center]
```

Comment:
There's some hesitancy between Camaron and Camoran, but Camoron is never used anywhere else:
ᐅ find . -iname '*.txt' -print0|xargs -0 grep Camaron|wc -l
60
ᐅ find . -iname '*.txt' -print0|xargs -0 grep Camoran|wc -l
18
ᐅ find . -iname '*.txt' -print0|xargs -0 grep Camoron|wc -l
0